src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,311 @@
+<!-- KEYWORDS: design pattern navigation drawer menu tab tabs navigate nav -->
 
-# Navigation Patterns
+# Navigation
 
-Navigation is the way in which users access the different areas of your application. This is typically accomplished through a side drawer navigation panel with a list of links to the various pages in your application.
+<ImageGrid images={[Banner]} regularWidth />
 
-<MaterialDesignDescription
-    title={'Understanding Navigation'}
-    url={'https://material.io/design/navigation/understanding-navigation.html#types-of-navigation'}
-    description={`Read about Material Design's navigation fundamentals.`}
+Navigation refers to how users access the different areas of your application and move from screen to screen. The most common forms of primary navigation are drawers and tabs.
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Understanding Navigation'}
+        url={'https://material.io/design/navigation/understanding-navigation.html#types-of-navigation'}
+        description={`Read about Material Design's navigation fundamentals.`}
+    />
+    <Divider />
+</React.Fragment>
+
+# Drawer Navigation
+
+A navigation drawer is a panel that is attached to the side of your application. These typically contain a [List](/patterns/lists) of primary pages in your application for the user to access quickly.
+
+<ImageGrid
+    images={[NavigationBreakdownImg]}
+    caption={['1. Header', '2. Subheader', '3. Body', '4. Footer']}
+    regularWidth
+/>
+
+Most drawers have four main sections:
+
+-   **Header**: contains the name of the company, name of the product, or a logo
+-   **Subheader**: an optional section that can be used to support custom content, such as filtering or search
+-   **Body**: contains a list of primary pages that users can navigate to
+-   **Footer**: contains the company logo, version number, and trademark information
+
+There are several different styles of drawer navigation, each with slightly different behavior.
+
+<React.Fragment>
+    <DemoCard
+        repository={'Drawer'}
+        angular={'https://pxblue-components.github.io/angular/?path=/story/components-drawer--within-a-drawer-layout'}
+        react={'https://pxblue-components.github.io/react/?path=/story/components-drawer--in-drawer-layout'}
+    />
+    <MaterialDesignDescription
+        title={'Navigation Drawer '}
+        url={'https://material.io/components/navigation-drawer/'}
+        style={{ marginRight: 0 }}
+    />
+</React.Fragment>
+
+## Permanent
+
+A permanent drawer is always visible and always open. The permanent drawer is typically full-width, as opposed to the rail variant (see below). Use this variation if you have:
+
+-   five or more top-level pages
+-   a nested navigation hierarchy
+-   enough space to justify it (typically desktop-only applications)
+
+<ImageGrid images={[NavigationPermanentImg]} caption={['A permanent drawer on desktop.']} regularWidth />
```

Comment:
Should the permanent drawer have an elevation, or should it be at the same level as the main content?

Reply:
I think we are trying to push for drawer w elevations for all variants